### DISCUSSION TITLE
Why is there no ansi_escape_codes parameter in base_connection.py

### DISCUSSION THREAD

**Discussion by Ziaf007** [ktbyers/netmiko#2691] [Q&A] [answered]
![Screenshot 2022-03-18 162848](https://user-images.githubusercontent.com/61587284/158991202-9fe33944-b993-4386-9dfa-028e77e346b7.png)

Why is the 'ansi_escape_codes' attribute hard-coded to a False?  Is that a mistake or done intentionally for some reason?
The issue with this is the following error in the console. As a Result of the attribute being False the strip_ansi_escape_codes() function does not get triggered and so the output Prompt remains bugged with Ansi_Codes and so the script can not recognise the Prompt.
The moment I turn the Value to True, the script works fine.

**Comment by ktbyers** [comment, marked as answer]
@Ziaf007 I had a bias towards not changing/removing things from the output unless I explicitly knew there were ANSI codes generated by the platform. 

But I might change that...

Anyways the near-term fix is just to enable it in the driver for the given platform or to pass it in as an argument explicitly.

**Reply by Ziaf007** [reply]
Okay got your point. Thank You.
